.github/workflows/cli.yml

```
@@ -116,3 +116,7 @@ jobs:
       - name: Run tests
         run: cargo test
         working-directory: ./functional-tests
+
+      - name: Run Formatting
+        run: cargo fmt
```

If you don't specify `--check`:
```suggestion
        run: cargo fmt --check
```
it will simply reformat and not output anything.

---

I think I've missed it. Thanks for pointing it out 👍.